tests/test_send.py

```diff
@@ -119,6 +119,10 @@ def test_email_dates(
         self, mocker: MockerFixture, app_fixture: FastAPI, full_user_mock: Mock
     ):
         client = TestClient(app_fixture)
+
+        mock_settings = mocker.patch("regtech_mail_api.internal.settings")
+        mock_settings.from_addr = "test@cfpb.gov"
+        mock_settings.beta_body_template = "\nCongratulations! This email confirms that {signer_name} submitted a filing on {formatted_date}. The confirmation number for this filing is {confirmation_id}.{line_break}The beta platform is for testing purposes only and user-supplied data may be removed at any time. Email our support staff at sblhelp@cfpb.gov to share feedback or return to the platform to upload a new file and continue testing.\n"
```

Comment:
think we can put these into the pyproject.toml `tool.pytest.ini_options` section, under the `env` area; and the template doesn't need to match the actual verbiage, just need to make sure those interpolation variables exist. I think the purpose is to make sure the sent email matches the formatted template, so the expected email can just be `settings.beta_body_template.format){...})` instead of the whole text.